lib/msal-core/src/utils/ErrorUtils.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AuthCache } from "../cache/AuthCache";
+import { AuthError } from "../error/AuthError";
+/**
+ * @hidden
+ */
+export class ErrorUtils {
+
+    static errorHandler(storage: AuthCache, errorFunc: AuthError): AuthError {
```

Comment:
If we are moving the error handlers from UserAgentApplication.ts to this class, then I agree we should put this here. If this is simply a function to reset errors and the storage items, we should move this to AuthError.ts as a static function and not create a new file.

Reply:
I want to eventually move all error handling here.

Reply:
Revised my stand, moved this to UAA as a helper function.